src/mrdParser.ts

```diff
@@ -2,78 +2,64 @@ import * as fs from 'fs';
 
 import { Participant } from './sequintence/participant';
 import { Relation } from './sequintence/relation';
+import { Diagram } from './types/diagram.type';
 
-export class MrdParser {
-  private participants: Participant[];
-  private relations: Relation[];
+type sourceAndTargetParticipants = {
```

Comment:
Please be consistent with naming convention
```suggestion
type SourceAndTargetParticipants = {
```